packages/core/test/unit/model/schema.test.ts

```diff
@@ -67,12 +63,12 @@ describe(`${Support.getTestDialectTeaser('Model')}Schemas`, () => {
       });
 
       it('should be able to override the default schema delimiter', () => {
-        expect(Company.schema(Company.table.schema, '^').table.delimiter).to.equal('^');
+        expect(Company.schema(Company.table.schema!, '^').table.delimiter).to.equal('^');
       });
 
       it('should support multiple, coexistent schema delimiter models', () => {
-        const schema1 = Company.schema(Company.table.schema, '$');
-        const schema2 = Company.schema(Company.table.schema, '#');
+        const schema1 = Company.schema(Company.table.schema!, '$');
```

Comment:
if we make `schema` accept `Nullish` instead of `null`, these non-nullish assertions won't be necessary

It's only a typing change, the implementation already accepts nullish values

Reply:
I already added the ! before updating the type, but that might be a nicer way to handle this

Reply:
Fixed